### PR TITLE
US S+ benefits tote bag offer

### PR DESCRIPTION
**Pull request by paul-daniel-dempsey**
## What are you doing in this PR?

Tote Bag Offer for US Supporter+ users ONLY. 

- Add Benefits for Tier2
- Benefits Bold Feature Tier2
- And bold card copy for Tier3
- TsAndCs contain minimum US S+ monthly & annual levels

Offer will be emailed afterwards (as per AUS).

[**Trello Card**](https://trello.com/c/hMVWeV9e/676-set-live-copy-change-for-us-landing-page-tote-benefit)

|Tier2|Tier3|TsAndCs|
|-----|-----|-----|
|image|image|image|

URL ->
https://support.thegulocal.com/us/contribute

### CODE REVIEW

File: support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx [hunk 1]
```diff
@@ -333,11 +342,14 @@ export function ThreeTierLanding(): JSX.Element {
...
+	const getCardContentBaseObject = (
+		cardTier: 1 | 2 | 3,
+		contributionTypeKeyOverride?: 'annual' | 'monthly',
+	) => {
 		const tierPlanCountryCharges =
-			tierCards[`tier${cardTier}`].plans[contributionTypeKey].charges[
-				countryGroupId
-			];
+			tierCards[`tier${cardTier}`].plans[
+				contributionTypeKeyOverride ?? contributionTypeKey
```
Comment (jamesgorrie) [inline support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx:351]: I get the logic, but it is quite complicated for what I think can just be hardcoded values?

**Review by jamesgorrie — APPROVED** [review]
Nice

File: support-frontend/assets/pages/supporter-plus-landing/components/threeTierDisclaimer.tsx [hunk 2]
```diff
@@ -50,3 +56,22 @@ export function ThreeTierDisclaimer({
 		</>
 	);
 }
+
+export function ToteTsAndCs({
+	currency,
+	toteCostMonthly,
+	toteCostAnnual,
```
[the remark this answers is not in the document]
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/supporter-plus-landing/components/threeTierDisclaimer.tsx:63]: Was just to be on safe-side, there was talk of tweaking prices on meeting earlier and not sure if this Tote offer could overrun into that.

File: support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx (hunk 1 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx:351]: Wanted to be on safe-side, there was talk of tweaking prices on meeting earlier and not sure if this Tote offer could overrun into that.

File: support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx [hunk 3]
```diff
@@ -50,8 +50,14 @@ import { navigateWithPageView } from 'helpers/tracking/ophan';
...
 import { SupportOnce } from '../components/supportOnce';
 import { ThreeTierCards } from '../components/threeTierCards';
-import { ThreeTierDisclaimer } from '../components/threeTierDisclaimer';
-import { tierCards } from '../setup/threeTierConfig';
+import {
+	ThreeTierDisclaimer,
+	ToteTsAndCs,
+} from '../components/threeTierDisclaimer';
+import {
+	tierCards as tierCardsNoTote,
```
[the remark this answers is not in the document]
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx:58]: Great spot, rushing, thats the proof of a  review, changed.

File: support-frontend/assets/pages/supporter-plus-landing/setup/threeTierConfig.ts [hunk 4]
```diff
@@ -295,6 +308,23 @@ const tier3: TierCard = {
...
 };
 
+const tier3tote: TierCard = {
+	...tier3,
+	benefits: {
+		description: [
+			'The rewards from All-access digital ',
+			{ strong: true, copy: 'including a limited-edition Guardian tote bag**' },
+		],
+		list: [...tier2.benefits.list],
```
[the remark this answers is not in the document]
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/supporter-plus-landing/setup/threeTierConfig.ts]: Great spot, rushing, thats the proof of a review, changed.

File: support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx:58]: I did this as kept the changes further down to a minimum (ie we use `tierCards` throughout this file).

**Review by jamesgorrie — APPROVED** [review]

**Pull request merged**